cdap-master-spi/src/main/java/io/cdap/cdap/master/spi/environment/spark/SparkConfig.java

```diff
@@ -60,4 +62,11 @@ public URI getSparkJobFile() {
   public Map<String, String> getConfigs() {
     return configs;
   }
+
+  /**
+   * Returns driver watcher thread
+   */
+  public Thread getSparkDriverWatcherThread() {
```

Comment:
instead of a Thread, it's better for this to be SparkDriverWatcher

Reply:
done